Mathlib/Data/NNRat/Defs.lean

```diff
@@ -13,8 +12,10 @@ import Mathlib.Data.Rat.Order
 /-!
 # Nonnegative rationals
 
-This file defines the nonnegative rationals as a subtype of `Rat` and provides its algebraic order
-structure.
+This file defines the nonnegative rationals as a subtype of `Rat` and provides its basic algebraic
+order structure.
+
+Note that `NNRat` is not declared as a `Field` here. See `Data.NNRat.Lemmas` for that instance.
```

Comment:
Given your motivation, I think you should add `#assert_not_exists Field` and `#assert_not_exists Semifield` to the bottom of this file, with a brief comment about a future `NNRat.cast`.

Reply:
(confusingly, the syntax is `assert_not_exists`, not `#assert_not_exists`!)